### PR TITLE
Adding Twilio SMS binding

### PR DESCRIPTION
**Pull request by fabiocav**

### CODE REVIEW

File: Resources/Resources.resx [hunk 1]
```diff
@@ -966,6 +966,45 @@
   <data name="timerTrigger_schedule_errorText" xml:space="preserve">
     <value>Invalid Cron Expression. Please consult the &lt;a target='_blank' href='https://azure.microsoft.com/en-us/documentation/articles/functions-bindings-timer/'&gt;documentation&lt;/a&gt; to learn more.</value>
   </data>
+    <data name="twilioSms_accountsid_help" xml:space="preserve">
```
Comment (fabiocav) [inline Resources/Resources.resx]: Only updated the language neutral resource file. I'm assuming the others are generated from this and we can properly fallback if the resource is not found in the culture specific resource files.

**Comment by fabiocav** [comment]
Also regenerated the `Resources.Designer.cs` file. Doesn't seem like it was required, since it was missing a lot of resources, but this brings it up to date.

**Comment by mathewc** [comment]
Looks OK to me. Looks like a lot of stuff has changed in templates (loc, etc.) so those guys should take a look as well.

File: Resources/Resources.resx [hunk 2]
```diff
@@ -966,6 +966,45 @@
...
+    <value>This is your Twilio account SID</value>
+  </data>
+  <data name="twilioSms_accountsid_label" xml:space="preserve">
+    <value>Account SID</value>
+  </data>
+  <data name="twilioSms_authtoken_help" xml:space="preserve">
+    <value>This is your Twilio Auth Token</value>
```
Comment (soninaren) [inline Resources/Resources.resx]: Nit: Missing "." at the end of line.

**Comment by soninaren** [comment]
We can put this behind filter like this and merge it in.

```JSON
"filters": [
                "showHiddenBinding"                
              ]
```
refer to this for more details
https://github.com/Azure/azure-webjobs-sdk-templates/issues/51

**Comment by soninaren** [comment]
Looks good.

**Comment by fabiocav** [comment]
Out of curiosity, what is the filter for?

**Comment by soninaren** [comment]
So that we can hide a binding in production until it is ready for release.

**Comment by fabiocav** [comment]
Technically, there's no reason to hide that binding. I'm assuming what we add here will be available in the next release (next tag), correct?

**Comment by mathewc** [comment]
Yeah, I don't think this needs to be hidden either. It just needs to be tagged along with the other changes like F# template updates, etc.

**Comment by soninaren** [comment]
Yeah we should not hide it.

**Pull request merged**